packages/blocks/src/components/button-spinner/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { Spinner } from '@wordpress/components';
```

Comment:
In general, we make an explicit effort not to have any dependencies from `@wordpress/components` inside `@crowdsignal/blocks`.   

Most of these require external stylesheets to be loaded for them to work nor do they match our own components or the theme in appearance.   
Even though it might not affect the spinner component itself, I'd rather not make a precedent - and a `Spinner` should be simple enough to reproduce on our end easily.

Reply:
I'm a bit confused here. We have a bunch of cool components inside the `@wordpress/components` but we can't use them to increment our blocks?
I agree that a Spinner is simple but, using it from the WP library would help us make the styles consistent. Wouldn't it?

Reply:
Right, but the problem is these are best considered 'WP Admin' components. As in, even if you write a WordPress theme or a plugin, you'd only want to use those for the editor/admin UI and not any of the public user-facing views where they would break.

Or think of it as a difference between a button component and a button block - like if we used `<Button>` from `@wordpress/components` instead of our button 'block', things wouldn't look great. So in the same way, we wouldn't want to use `@crowdsignal/components` here either.  
Additionally, for some of these - especially the interactive ones - we might need to provide custom behaviors anyway so they work for our use case.

So while it could be made to work, we opted against it in the hope it would allow us to keep the bundle size smaller as well... which to be honest, I'm not sure if it actually helped much unfortunately.

Reply:
I see, thanks for clarifying! 👍